src/components/admin/form-components/ListSelector.jsx

```diff
@@ -37,8 +37,13 @@ export default class ListSelector extends React.Component {
 }
 
 ListSelector.propTypes = {
-  elements: PropTypes.object.isRequired,
-  type: PropTypes.string,
+  elements: PropTypes.arrayOf(
```

Comment:
I like this change a lot, an array is definitely the way to go lol, as you said, you were designing for the implementation haha